nexus_motion_planner/test/test_request.test.py

```diff
@@ -318,6 +355,8 @@ def generate_test_description():
         trigger_lifecycle_5,
         trigger_lifecycle_6,
         trigger_get_plan_irb1300_poses,
+        ros2_control_node,
+        *load_controllers,
```

Comment:
What does the `*` do? Is it a pythonic way to append to a list?

Reply:
The `*` unpacks the list into its composite elements (you can think of it as an `extend()` but in the middle of the list.

It was introduced to avoid "clunky syntax handling". https://peps.python.org/pep-3132/ , so I think it's acceptable.

The technically simpler way to do it would be:
```

    return launch.LaunchDescription([
        robot_state_publisher,
        motion_planner,
        move_group_node,
        static_tf_node_1,
        static_tf_node_2,
        trigger_lifecycle_1,
        trigger_lifecycle_2,
        trigger_lifecycle_3,
        trigger_lifecycle_4,
        trigger_lifecycle_5,
        trigger_lifecycle_6,
        trigger_get_plan_irb1300_poses,
        ros2_control_node]
  + load_controllers
  + [launch_testing.actions.ReadyToTest()
    ]), node_mapping
```

But I didn't like the triple addition.